modules/initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="initiating-limited-live-migration_{context}"]
+= Initiating the limited live migration process
+
+After you have checked the behavior of EgressIP resources, EgressFirewall resources, and multicast enabled namespaces, and removed EgressRouter resources, you can initiate the limited live migration process.
+
+.Prerequisites
+
+* A cluster has been configured with the OpenShift SDN CNI network plugin in the network policy isolation mode.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a recent backup of the etcd database.
+* The cluster is in a known good state without any errors.
+* Before migration to OVN-Kubernetes, a security group rule must be in place to allow UDP packets on port `6081` for all nodes on all cloud platforms.
+* You have patched the `100.64.0.0/16` and `100.88.0.0/16` if they were previously in use by OpenShift-SDN.
+* You have checked for EgressIP resources, EgressFirewall resources, and multicast enabled namespaces.
+* You have removed any egress router pods before beginning the limited live migration. For more information about egress router pods, see "Deploying an egress router pod in redirect mode".
+* You have reviewed the "Considerations for limited live migration to the OVN-Kubernetes network plugin" section of this document.
+
+.Procedure
+
+ifdef::openshift-rosa,openshift-dedicated[]
+. Enter the following command to add the `unsupported-red-hat-internal-testing` annotation to the cluster-level network configuration:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster --type='merge' --patch '{"metadata":{"annotations":{"unsupported-red-hat-internal-testing": "true"}}}'
```

Comment:
this step no need for un-managed cluster.

Reply:
This step does not show up for unmanaged clusters--it will only show up in ROSA clusters when we include the docs over there.

Reply:
ok,thanks